src/shared-components/dropdown/index.js

```diff
@@ -7,7 +7,10 @@ import allowNullPropType from '../../utils/allowNullPropType';
 
 const defaultProps = {
   textAlign: 'left',
+  /* eslint-disable  @typescript-eslint/no-empty-function */
   onChange() {},
+  maxHeight: '250px',
+  value: null,
```

Comment:
[Nit] Did you see any issues on your end with this as `undefined` instead of `null`? 

In all likelihood it probably does not matter, but sometimes 3rd party libraries (especially in Type definitions) draw a meaningful distinction, so it might be preferable to keep this as `undefined` (which is what current behavior is, if not provided) instead of null.